src/realm/spec.cpp

```diff
@@ -250,34 +243,6 @@ void Spec::erase_column(size_t column_ndx)
     update_internals();
 }
 
-void Spec::upgrade_string_to_enum(size_t column_ndx, ref_type keys_ref)
-{
-    REALM_ASSERT(get_column_type(column_ndx) == col_type_String);
-
-    // Create the enumkeys list if needed
-    if (!m_enumkeys.is_attached()) {
-        m_enumkeys.create(Array::type_HasRefs, false, m_num_public_columns);
-        m_top.set(4, m_enumkeys.get_ref());
```

Comment:
what should we do with `Spec::m_enumkeys` and `Spec::s_enum_keys_ndx` ? I think they can be removed as nobody should be using them, but maybe we should also assert that there is nothing stored there?

Reply:
That was an oversight. Should be removed.